contracts/WrappedTokenBridgeERC20Upgradable.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {LzLib} from "@layerzerolabs/solidity-examples/contracts/libraries/LzLib.sol";
+import {WrappedTokenBridgeUpgradable} from "./WrappedTokenBridgeUpgradable.sol";
+import {IBridgedERC20} from "./interfaces/IBridgedERC20.sol";
+
+/// @dev Mints a wrapped token when a message received from a remote chain and burns a wrapped token when bridging to a remote chain
+contract WrappedTokenBridgeERC20Upgradable is WrappedTokenBridgeUpgradable {
```

Comment:
I'm not sure that burn/burnFrom is part of ERC20Upgradable
I think it's in ERC20Burnable only